src/test/java/edu/harvard/iq/dataverse/externaltools/ExternalToolServiceBeanTest.java

```diff
@@ -338,7 +338,7 @@ public void testParseAddExternalToolInputWrongType() {
         }
         assertNotNull(expectedException);
         System.out.println("exception: " + expectedException);
-        assertEquals("Type must be one of these values: [explore, configure, preview].", expectedException.getMessage());
+        assertEquals("Type must be one of these values: [explore, configure, preview, query].", expectedException.getMessage());
```

Comment:
The API tests failed with this from https://jenkins.dataverse.org/job/IQSS-Dataverse-Develop-PR/job/PR-9737/2/consoleFull

RUNNING HANDLER [dataverse : enable and restart payara] ************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Unable to start service payara: Job for payara.service failed because the control process exited with error code.\nSee \"systemctl status payara.service\" and \"journalctl -xe\" for details.\n"}

We should make sure they're passing before moving to QA.

Reply:
All tests are passing now.